app/tests/tests_10/test_validators.py

```diff
@@ -1,55 +1,15 @@
 from django.core.exceptions import ValidationError
 from django.test import TestCase
 
-from rest_framework import serializers
-
 from stac_api.validators import MediaType
 from stac_api.validators import get_media_type
 from stac_api.validators import normalize_and_validate_media_type
 from stac_api.validators import validate_content_encoding
 from stac_api.validators import validate_item_properties_datetimes_dependencies
-from stac_api.validators_serializer import validate_asset_href_path
-
-from tests.tests_10.data_factory import Factory
 
 
 class TestValidators(TestCase):
 
-    def test_validate_asset_href_path(self):
-        factory = Factory()
-        collection = factory.create_collection_sample(name='collection-test').model
-        item = factory.create_item_sample(collection=collection, name='item-test').model
-        validate_asset_href_path(item, 'asset-test', 'collection-test/item-test/asset-test')
-
-        with self.settings(AWS_S3_CUSTOM_DOMAIN='new-domain'):
-            validate_asset_href_path(item, 'asset-test', 'collection-test/item-test/asset-test')
-
-        with self.settings(AWS_S3_CUSTOM_DOMAIN='new-domain/with-prefix/'):
-            validate_asset_href_path(
-                item, 'asset-test', 'with-prefix/collection-test/item-test/asset-test'
-            )
-
-        with self.settings(AWS_S3_CUSTOM_DOMAIN='//new-domain/with-prefix'):
-            validate_asset_href_path(
-                item, 'asset-test', 'with-prefix/collection-test/item-test/asset-test'
-            )
-
-        with self.settings(AWS_S3_CUSTOM_DOMAIN='//new domain/with-prefix'):
-            validate_asset_href_path(
-                item, 'asset-test', 'with-prefix/collection-test/item-test/asset-test'
-            )
-
-        with self.assertRaises(
-            serializers.ValidationError,
-            msg="Invalid Asset href path did not raises serializers.ValidationError"
-        ):
-            validate_asset_href_path(item, 'asset-test', 'asset-test')
-            validate_asset_href_path(item, 'asset-test', 'item-test/asset-test')
-            validate_asset_href_path(item, 'asset-test', 'collection-test/item-test/asset-test')
-            validate_asset_href_path(
-                item, 'asset-test', '/service-stac-local/collection-test/item-test/asset-test'
-            )
-
```

Comment:
Hmm, are you sure this check aren't useful at all? (together with the corresponding validate functions)

Reply:
I'll check again. I had the impression that there were functions that were only called in the tests, but there might be some other magic going on, see comment above.

Reply:
cc @ltshb